Formula/e/easyrpg-player.rb

```diff
@@ -35,6 +35,13 @@ class EasyrpgPlayer < Formula
   depends_on "sdl2"
   depends_on "speexdsp"
 
+  uses_from_macos "expat"
```

Comment:
Looks like this actually needs to be an unconditional dependency

Reply:
that actually makes sense.